scene/resources/world_2d.cpp

```diff
@@ -211,7 +211,7 @@ struct SpatialIndexer2D {
 			List<VisibilityNotifier2D *> added;
 			List<VisibilityNotifier2D *> removed;
 
-			int visible_cells = (end.x - begin.x) * (end.y - begin.y);
+			uint64_t visible_cells = static_cast<uint64_t>(end.x - begin.x) * static_cast<uint64_t>(end.y - begin.y);
```

Comment:
We prefer using C-style casts:
```suggestion
			uint64_t visible_cells = (uint64_t)(end.x - begin.x) * (uint64_t)(end.y - begin.y);
```

Reply:
I've amended the commit